README.md

```diff
@@ -68,10 +68,7 @@ Open terminal from your computer
 
 `- GET /trips/myRequest - view all my request history`
 
-`- GET /user/profile - get user profile information`
```

Comment:
@Niyonsengaeric  Any reasons why you removed this lines?

Reply:
sorry!